models/prostate_mri_anatomy/scripts/center_crop.py

```diff
@@ -0,0 +1,71 @@
+import argparse
```

Comment:
Hi @wyli ,

Do we need any copyright in the python scripts of bundle?
I am not very sure about it, what do you think?

Thanks.

Reply:
Some other scripts also miss a copyright. E.g., 

https://github.com/Project-MONAI/model-zoo/blob/dev/models/brats_mri_segmentation/scripts/prepare_datalist.py
https://github.com/Project-MONAI/model-zoo/blob/dev/models/mednist_gan/scripts/losses.py
https://github.com/Project-MONAI/model-zoo/blob/dev/models/pancreas_ct_dints_segmentation/scripts/prepare_datalist.py

However, some have a copyright: 
https://github.com/Project-MONAI/model-zoo/blob/dev/models/pancreas_ct_dints_segmentation/scripts/search.py
https://github.com/Project-MONAI/model-zoo/blob/dev/models/wholeBrainSeg_Large_UNEST_segmentation/scripts/networks/nest_transformer_3D.py
https://github.com/Project-MONAI/model-zoo/blob/dev/models/wholeBrainSeg_Large_UNEST_segmentation/scripts/networks/patchEmbed3D.py

I've added a Copyright to the file and can also add one to the other scripts in a new PR (if desired)

Reply:
One of these files is in the mednist_gan bundle I put up, it's such a tiny amount of trivial code I'm not worried about its license status.